docs/rfc/ethereum_attestation.md

```diff
@@ -0,0 +1,129 @@
+# RFC: Ethereum attestation
+
+* Author: @CodeSandwich
+* Date: 2021-03-05
+* Status: draft
+* Community discussion: n/a
+
+## Motivation
+
+The attestation between Link and Ethereum is a valuable building block for a user identity.
+It brings the Link reputation coming from projects and contributions to the
+Ethereum world of DAOs and donations, where it's important to know who's behind an address.
+On the other hand, it lends Ethereum account reputation with its assets and undeniable history
+to Link to build user's trust in the identity.
+
+## Overview
+
+This RFC is built on top of [identities spec][identities].
+It introduces support for Ethereum address claims on Link
+and a smart contract on Ethereum to make Link identity claims on Ethereum.
+
+## Link identity JSON extension
+
+The identity JSON `payload` structure supports a new key: `https://radicle.xyz/ethereum/claim/v1`.
+Under this key there is stored an ethereum address claim following this convention:
+
+- `address` - the claimed ethereum address, encoded according to [EIP-55][eip-55],
+e.g. using [ethers.js][ethers-addr]
+- `expiration` - the claim expiration timestamp, encoded as a [JavaScript Date][date]
+
+Example:
+```json
+{
+    "payload": {
+        "https://radicle.xyz/ethereum/claim/v1": {
+            "address": "0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B",
+            "expiration": "2021-03-19T23:15:30.000Z"
+        }
+        ...
+    }
+    ...
+}
+```
+
+## Ethereum smart contract
+
+A new Ethereum smart contract is deployed to the network, which lets users claim their Radicle IDs:
+
+```solidity
+contract Claims {
+    event Claimed(address indexed addr);
+    function claim(uint8 version, uint256 id) public;
+}
+```
+
+To claim an ID, call `claim` using your Ethereum account.
+It will emit an event `Claimed`, which later can be queried to discover your attestation.
+The claims have no expiration date and don't need to be renewed.
+
+Every new claim invalidates previous ones made with the same account.
+To only revoke a claim without creating a new one, use ID `0`,
+which is guaranteed to not match any existing identity.
+
+Currently supported `version` values:
```

Comment:
Is `version` supposed to be in the payload above? Or where did it come from?

Reply:
Or would you use `claim/v1` and `claim/v2` URLs?

Reply:
The payload is the Link-stored data, the `version` is used only in Ethereum. It has no corelation with the Link payload URL versioning.

Reply:
Ah, I missed it in the contract function :)